src/app/main/image-pathes/places-icons.ts

```diff
@@ -0,0 +1,12 @@
+export const placesIcons = {
+  redIcon: 'assets/img/places/red-marker.png',
+  greenIcon: 'assets/img/places/green-marker.png',
+  bookmark: 'assets/img/places/bookmark-default.svg',
+  bookmarkSaved: 'assets/img/places/bookmark-set.svg',
+  star: 'assets/img/places/star-1.png',
+  starHalf: 'assets/img/places/star-filled-half.png',
+  starUnfilled: 'assets/img/places/star-2.png',
+  searchIcon: 'assets/img/places/search.svg',
+  notification: 'assets/img/places/notification-unread.svg',
+  share: 'assets/img/places/share.svg'
+};
```

Comment:
let's export each one separately, so there will. be no need to create variables

Reply:
Done, but left some for the HTML interpolation purpose